### PR TITLE
Issue 17633 contact us page not render

### PR DESCRIPTION
**Pull request by alfredo-dotcms**

### CODE REVIEW

File: components/content-types/Forms/Forms.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+const DotFormWC = reactifyWc('dot-form');
+
+if (process.browser) {
+    import('dotcms-field-elements/dist/loader').then((module) => {
+        module.defineCustomElements(window);
+    });
+}
+
+const Forms = ({ layout, variable }) => {
```
Comment (fmontes) [inline components/content-types/Forms/Forms.js]:
```suggestion
const Form = ({ layout, variable }) => {
```

File: components/content-types/Forms/FormsWidget.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+                .form({ identifier: formId })
+                .get()
+                .then(({ fields }) => {
+                    const layout = fields.map((item) => {
+                        return { columns: [{ fields: [item] }] };
+                    });
+                    setItems(layout);
+                });
+        }
+    });
```
Comment (fmontes) [inline components/content-types/Forms/FormsWidget.js]:
```suggestion
    }, [formId]);
```

File: components/content-types/Forms/FormsWidget.js [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+import PropTypes from 'prop-types';
+import Forms from './Forms';
+const dotCMSApi = require('../../../utils/dotcms/dotcmsApi');
+
+const FormsWidget = ({ formId, rendered }) => {
+    const [items, setItems] = useState([]);
+    let variable;
+
+    if (rendered) {
+        variable = rendered.match(/variable='[^']+('*)/g)[0].split("'")[1];
```
Comment (fmontes) [inline components/content-types/Forms/FormsWidget.js]: Can't we get this from the contentlet object?

Comment (alfredo-dotcms) [reply components/content-types/Forms/FormsWidget.js]: no, this only comes in the `rendered` attr

File: components/content-types/Form/FormWidget.js [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+
+    if (rendered) {
+        variable = getFormVariableFromContent(rendered);
+    }
+
+    useEffect(() => {
+        dotCMSApi
+            .form({ identifier: formId })
+            .get()
+            .then(({ fields }) => {
```
Comment (fmontes) [inline components/content-types/Form/FormWidget.js]: Here we can get the variable name:

Screen Shot 2019-12-10 at 10 40 21

**Pull request merged**